main.js

```diff
@@ -37,6 +37,7 @@ function createWindow() {
         {
             label: 'Edit',
             submenu: [
+                {role: 'reload', accelerator: 'CmdOrCtrl+R', selector: 'reload:'},
```

Comment:
Could this be capitalized?
```suggestion
                {role: 'Reload', accelerator: 'CmdOrCtrl+R', selector: 'reload:'},
```

Reply:
Yes, it's case-insensitive.